plugins/inputs/win_perf_counters/win_perf_counters.go

```diff
@@ -265,6 +265,11 @@ func (m *Win_PerfCounters) Gather(acc telegraf.Accumulator) error {
 					} else if metric.instance == s {
 						// Catch if we set it to total or some form of it
 						add = true
+					} else if strings.Contains(metric.instance, "#") && strings.HasPrefix(metric.instance, s) {
```

Comment:
No this funktion is iterating over all process that are retrieved for the specific counter instances